hw/ip/otbn/data/base-insns.yml

```diff
@@ -357,7 +357,7 @@
 - mnemonic: csrrs
   rv32i: true
   synopsis: Atomic Read and Set bits in CSR
-  operands: [grd, csr, grs1]
+  operands: [grd, csr, grs]
```

Comment:
We chose to keep the RISC-V naming for single-register instructions (@imphil: I think this was your choice?), so this isn't the right fix. The correct thing is to update the name in `insn.py`.

Reply:
yes, that's correct.